Content.Client/Audio/ContentAudioSystem.AmbientMusic.cs

```diff
@@ -72,14 +76,15 @@ private void InitializeAmbientMusic()
         SubscribeNetworkEvent<RoundEndMessageEvent>(OnRoundEndMessage);
     }
 
-    private void AmbienceCVarChanged(float obj)
+    private void AmbienceCVarChangedAmbientMusic(float obj)
     {
         _volumeSlider = SharedAudioSystem.GainToVolume(obj);
 
         if (_ambientMusicStream != null && _musicProto != null)
-        {
             _audio.SetVolume(_ambientMusicStream, _musicProto.Sound.Params.Volume + _volumeSlider);
-        }
+
+        if (_ambientLoopStream != null && _loopProto != null)
+            _audio.SetVolume(_ambientLoopStream, _loopProto.Sound.Params.Volume + _volumeSlider);
```

Comment:
Looks like `_ambientLoopStream` never gets shut down, so the audio entity will leak. That's why the tests were failing before (they pass now because there's no test for entity leaks on planet maps).

Right below here in the code is the method that shuts down the ambient music stream. Either piggyback on it or do what it does and make sure it gets called in `ContentAudioSystem.Shutdown` (line 70)

Reply:
![image](https://github.com/space-wizards/space-station-14/assets/96445749/34f177d5-4215-49b3-95b8-9e4cb971bdcd)
![image](https://github.com/space-wizards/space-station-14/assets/96445749/86991390-4d7b-4678-b00e-265e919e9954)

it seems to me that FadeOut() automatically deletes entities whose volume has dropped to 0